moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -43,6 +43,9 @@
 #include <mutex>
 
 #include <std_msgs/msg/bool.h>
+// Optionally schedule a thread deadline for closer-to-realtime performance
+#include <linux/sched.h>
```

Comment:
We just had a discussion today about MoveIt 2 on Windows, does this make Windows support complicated?

Reply:
yes...  :/

Reply:
How about wrapping this stuff in a preprocessor directive to check if it's on a Unix system?